.github/workflows/release-notification.yaml

```
@@ -0,0 +1,33 @@
+# This action sends a notification to discord about new releases
+
+name: Release notification
+
+# Controls when the workflow will run
+on:
+  release:
+    types: [published]
+
+jobs:
+  test_message:
+    runs-on: ubuntu-latest
+    
+    # Skip in case it's a prerelease
+    if: "!github.event.release.prerelease"
+
+    steps:
+
+      # Get the fields for the release
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.2.3
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+    
```

```suggestion
      # Get the fields for the release
      - name: Get release
        id: get_release
        uses: bruceadams/get-release@v1.2.3
    
```

IMO not needed the release is public